examples/misc_controls_deviceorientation.html

```diff
@@ -13,13 +13,30 @@
 			photo by <a href="http://www.flickr.com/photos/jonragnarsson/2294472375/" target="_blank" rel="noopener">Jón Ragnarsson</a>.
 		</div>
 
+		<button id="deviceBtn">Open device orientation(Only in IOS 13+)</button>
 		<script type="module">
 
 			import * as THREE from '../build/three.module.js';
 
 			import { DeviceOrientationControls } from './jsm/controls/DeviceOrientationControls.js';
 
 			var camera, scene, renderer, controls;
+			var deviceBtn = document.getElementById("deviceBtn");
+
+			if ( window.DeviceOrientationEvent === undefined || typeof window.DeviceOrientationEvent.requestPermission !== 'function' ) {
+
+				document.body.removeChild( deviceBtn );
+
+			} else {
+
+				deviceBtn.addEventListener( 'click', () => {
+
+					document.body.removeChild( deviceBtn );
+					controls.connect();
```

Comment:
`.connect()` is also called in the ctor of `DeviceOrientationControls`. TBH, it's a bit hacky calling it called twice.

Developing the example according to https://github.com/mrdoob/three.js/pull/17823/files#r341047206 should make this code obsolete.